source/taup/taup_time.rst

```diff
@@ -31,7 +31,8 @@ TauP 默认台站位于地表，可使用 ``--stadepth`` 选项设置台站深
 
    $ taup_time -h 300 -evt 40 10 -sta -20 -10 -ph P,S,PcP,ScS,PKiKP
 
-在震中距比较小的情况下，可以使用 ``-km`` 选项指定震中距（单位为 km）。
+在震中距比较小的情况下，可以使用 ``-km`` 选项指定震中距（单位为 km）,
+此时 taup_time 假定地球半径为 6371 km，并不以实际参考模型的半径为准。
```

Comment:
I think we all agree that it's a TauP bug.

As the km to degree conversion is done when parsing the arguments, the model information may not be available at that time. So I'm not sure if TauP will fix it or not.

I think we should change it to a **note** or **warning**, and add a link to the issue https://github.com/crotwell/TauP/issues/12.

Reply:
Good~